model/facade/action/GetProjectUserWeeklyHoursReportAction.php

```diff
@@ -0,0 +1,127 @@
+<?php
+/*
+ * Copyright (C) 2009 Igalia, S.L. <info@igalia.com>
+ *
+ * This file is part of PhpReport.
+ *
+ * PhpReport is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * PhpReport is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with PhpReport.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+
+/** File for GetProjectUserWeeklyHoursReportAction
+ *
+ *  This file just contains {@link GetProjectUserWeeklyHoursReportAction}.
+ *
+ * @filesource
+ * @package PhpReport
+ * @subpackage facade
+ * @author Tony Thomas <tthomas@igalia.com>
+ */
+include_once(PHPREPORT_ROOT . '/model/facade/action/Action.php');
+include_once(PHPREPORT_ROOT . '/model/dao/DAOFactory.php');
+
+/** Get Users Weekly Hours Report Action
+ *
+ *  This action is used for retrieving information about work done on a given Project
+ *  by all Users per week.
+ *
+ * @package PhpReport
+ * @subpackage facade
+ * @author Tony Thomas <tthomas@igalia.com>
+ */
+class GetProjectUserWeeklyHoursReportAction extends Action{
+
+	/** The date interval init.
+	 *
+	 * This variable contains the initial date of the interval whose Tasks information we want to retrieve.
+	 *
+	 * @var DateTime
+	 */
+	private $init;
```

Comment:
Watch out the configuration of your IDE, this file is indented with tabs instead of spaces.